src/open_inwoner/scss/views/App.scss

```diff
@@ -192,7 +192,7 @@
   /// Font.
 
   --font-color-heading: var(--color-black);
-  --font-family-heading: 'TheMix C5';
+  --font-family-heading: 'Heading';
```

Comment:
Since right now (almost)( all H1's are defined by the design-tokens submodule, they are not overwritten with the font-file upload, so add a fallback font to the H1 fiel in the .utrecht-heading-1 class, like `font-family: var(--font-family-heading-1), var(--font-family-heading-1) ;` - which hopefully is enough to overwrite this.